dev-tools/validation/check-authz-route-matrix.py

```
@@ -114,6 +380,1261 @@ def validate_auth_path_vocabulary(document: dict[str, Any], errors: list[str]) -
+
+    token_predicates = (token_type, token_issuer)
+    if token_predicates != next(iter(type_issuer_pairs)):
+        errors.append(
+            f"{label} multi-profile token predicates must match the shared token_type/token_issuer"
+        )
+
+
+def validate_pending_deletion_generation(
+    route: dict[str, Any],
+    label: str,
+    account_generation: Any,
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `account_generation`

(ANN401)